test/helpers/db.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash');
+const { Pool } = require('pg');
+
+const {
+  testing: { username },
+  testing: dbConfig
+} = require('../../config/db');
+
+const usersTable = 'users';
+const testDbConfig = { ...dbConfig, user: username };
+
+exports.cleanRecord = (response, removeId = false) => {
```

Comment:
Be more specific in the method name, maybe you can re name as `removeObjectFields` ? 

Also, maybe we can pass to the function the fields to remove. Something like: 

```
Function: 

exports.removeObjectFields = (object, fieldsToRemove) =>  _.omit(object, fieldsToRemove);

Calling function example:

const myObj = {
    id: 1,
    name: 'Dani',
    email: 'dani@dani.dani'
    created_at: '2021-05-05',
    updated_at: '2021-05-05'
};
const objectCleaned = removeObjectFields(myObj, ['id', 'created_at', 'updated_at']);